docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -36,18 +70,19 @@ include::{tab-widgets}/download-widget.asciidoc[]
 // end::install-elastic-agent[]
 --
 
-. From the agent directory, run the appropriate command to install {agent} as
-a managed service, enroll it in {fleet}, and start the service. Don't have a
-{fleet} enrollment key? Read the
-{fleet-guide}/fleet-quick-start.html[Quick start guide] to learn how to get one
-from {fleet}.
+. From the agent directory, run the following commands to install {agent},
+enroll it in {fleet}, and start it as a service.
++
+NOTE: On macOS, Linux (tar package), and Windows, run the `install` command to
+install {agent} as a managed service, enroll it in {fleet}, and start the
+service. The DEB and RPM packages include a service unit for Linux systems with
+systemd, so use the `enroll` command instead of `install`.
```

Comment:
In a coming PR, with the help of Ruflin and team, we need to document the Docker usage... unless we want to keep that primarily in the Beats / Fleet-Server repo (just for now, due to time).  We know it is used with Kubernetes and on its own and there are significant usage options to document.  A separate topic is likely appropriate regardless

Reply:
Yup, that work is being tracked in https://github.com/elastic/observability-docs/issues/622

Reply:
Also noting that we just merged a PR that hooked up the docs about running on K8s.